remind.py

```diff
@@ -118,10 +118,16 @@ def run_remind(config, blockers, server, header):
 					smtp.ehlo_or_helo_if_needed()
 
 					# send email to all addresses
-					smtp.sendmail(msg['From'], msg['To'], msg.as_string())
+					response = smtp.sendmail(msg['From'], msg['To'], msg.as_string())
+
+					# log success if all recipients recieved reminder, otherwise raise exception
+					if response == {}:
+						print("Reminder successfully accepted by mail server for delivery")
+					else:
+						raise Exception("Mail server cannot deliver reminder to following recipients: {}".format(response))
```

Comment:
did you mean to log msg['To']

Reply:
The `response` var would be the following:

> If this method does not raise an exception, it returns a dictionary, with one entry for each recipient that was refused. Each entry contains a tuple of the SMTP error code and the accompanying error message sent by the server.

So this logs every recipient who is not expected to receive the report/reminder

Reply:
ah, okay. Then it makes sense.